.changeset/replace-dead-link.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
@soudai-s please remove this file, a changeset is not necessary for website updates 👀

Reply:
Thank you for the comment. I deleted it :)